content/geometry/object-environment/environment-structure.adoc

```diff
@@ -0,0 +1,482 @@
+= Model structure
+
+== Structure overview
+
+[plantuml]
+----
+legend
+[[../geometry/object-classes.html#_environment_t Environment_Root (T)]]
+|_[[../geometry/object-classes.html#_terrain_t Terrain]]
+|_[[../geometry/object-classes.html#_environment_objects_t Objects]]
+  |_ [[../geometry/object-classes.html#_buildings Buildings]]
+  |_ [[../geometry/object-classes.html#_vegetation Vegetation]]
+  |_ [[../geometry/object-classes.html#_other_objects Other_Objects]]
+|_[[../geometry/object-classes.html#_road_network_t Road_Network]]
+  |_ [[../geometry/object-classes.html#_drivable_area_t Drivable_Area]]
+  |_ [[../geometry/object-classes.html#_sidewalk_t Sidewalk]]
+  |_ [[../geometry/object-classes.html#_road_marks_t Road_Marks]]
+  |_ [[../geometry/object-classes.html#_road_object_idx_t Road_Object_<object_idx> (T)]]
+  |_ [[../geometry/object-classes.html#_signal_signal_idx_t Signal_<signal_idx> (T)]]
+    |_ [[../geometry/object-classes.html#_sign_idx_t Sign_<sign_idx> (T)]]
+    |_ [[../geometry/object-classes.html#_light_idx_t Light_<light_idx> (T)]]
```

Comment:
The Traffic_Light_<traffic_light_idx> group node is missing as a parent for the Light and Cover group nodes. Makes it easier to understand the Light and Cover node groups on the first look and to select all objects of traffic lights at once.

Other possibility would be to name them Traffic_Lights_Lights and Traffic_Lights_Cover.